src/main/java/com/icthh/xm/tmf/ms/activation/domain/SagaTransaction.java

```diff
@@ -16,12 +16,14 @@
 import javax.persistence.Table;
 import javax.validation.constraints.NotNull;
 import lombok.Data;
+import lombok.ToString;
 import lombok.experimental.Accessors;
 import org.hibernate.annotations.GenericGenerator;
 
 @Data
 @Entity
 @Table(name = "saga_transaction")
+@ToString(exclude = "context")
```

Comment:
pls add log that write context on creation, for ability to find event id by context

Reply:
let's continue discussion in tread above